python/pyspark/ml/feature.py

```diff
@@ -6053,16 +6852,16 @@ def __init__(
     def setParams(
         self,
         *,
-        numTopFeatures=50,
-        featuresCol="features",
-        outputCol=None,
-        labelCol="labels",
-        selectorType="numTopFeatures",
-        percentile=0.1,
-        fpr=0.05,
-        fdr=0.05,
-        fwe=0.05,
-    ):
+        numTopFeatures: int = 50,
+        featuresCol: str = "features",
+        outputCol: Optional[str] = None,
+        labelCol: str = "label",
```

Comment:
This is probably OK, as that's the default in constructor and on JVM side, but shouldn't be added without indicating the change.